ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts

```diff
@@ -48,3 +48,49 @@ export interface MhrRegistrationIF {
     }
   ]
 }
+
+export interface NewMhrRegistrationApiIF {
+  documentId: string
+  clientReferenceId?: string
+  declaredValue?: string
+  submittingParty: {
+    personName?: {
+      first: string
+      last: string
+      middle?: string
+    }
+    businessName?: string
+    address: {
+      street: string
+      city: string
+      region?: string
+      country?: string
+      postalCode?: string
+    }
+    emailAddress?: string
+    phoneNumber?: string
+    phoneExtension?: string
+  }
+  ownerGroups: MhrRegistrationHomeOwnerGroupIF[]
+  // owners are only included in search detail results.
+  // owners: MhrRegistrationHomeOwnersIF[]
```

Comment:
Is this property meant to be commented out? 
In development?

Reply:
it can be removed, especially the second line, kept it there to let people know why this isn't even optional.

Reply:
removed!